cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -59,7 +58,8 @@ export class PassportAuth implements Auth {
    * @param sessionOpts - Session options to be used by express-session
    */
   public init(app: express.Express, sessionOpts: SessionOptions) {
-    this.log.info('Initializing express app to use express session and passport');
+    logger.info('Initializing express app to use express session and passport',
```

Comment:
by default logger is off, need getter and setter to instantiate would do this at a demo level , no need for getting and setting at module level.